bigtable/autoscaler/strategies.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2017 Google Inc.
```

Comment:
The idea is we are going to add a few alternative strategies such as exponential, and have them all organized in here.